### DISCUSSION TITLE
[BUG] 1Panel登录瞬间 CPU 占用过高的问题

### DISCUSSION THREAD

**Discussion by Anyexyz** [1Panel-dev/1Panel#4342] [Q&A] [answered]
### 联系方式

anyexyz@foxmail.com

### 1Panel 版本

v1.10.2-lts

### 问题描述

1Panel登录瞬间 CPU 占用过高的问题

### 重现步骤

1Panel 登录后，主页状态显示CPU占用率通常会显示一个较高的数值，等下一次刷新状态的时候才会回落

### 期待的正确结果

正常输出CPU占用状态，减轻登录时的占用

### 附加信息

![13a7408c51f559caa4f84f745380435](https://github.com/1Panel-dev/1Panel/assets/53684988/47f44d83-78a9-432a-b2de-e878f800f221)

**Comment by ssongliu** [comment, marked as answer]
感谢反馈，这里主要是获取概览界面数据占用的资源，登陆的瞬间需要去获取两部分数据：
1. 基础参数，包括主机信息、cpu 、内存、磁盘等总量；
2. 瞬时数据，包括当前 cpu 、内存磁盘网络等实时占用；

登陆的瞬间会获取 1.2 所有数据，然后后面定时刷新就只去请求 2 了。
